source/wp-content/themes/wporg-themes-2024/src/theme-downloads/index.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Block Name: Downloads Chart
+ *
+ * @package wporg
+ */
+
+namespace WordPressdotorg\Theme\Theme_Directory_2024\Theme_Downloads_Block;
+
+defined( 'WPINC' ) || die();
+
+add_action( 'init', __NAMESPACE__ . '\init' );
+
+/**
+ * Register the block.
+ */
+function init() {
+	register_block_type(
+		dirname( dirname( __DIR__ ) ) . '/build/theme-downloads',
+		array(
+			'render_callback' => __NAMESPACE__ . '\render',
+		)
+	);
+
+	// phpcs:ignore WordPress.WP.EnqueuedResourceParameters.MissingVersion -- third-party script.
+	wp_register_script( 'google-charts-loader', 'https://www.gstatic.com/charts/loader.js', array(), null, true );
+}
+
+/**
+ * Render the block content.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ *
+ * @return string Returns the block markup.
+ */
+function render( $attributes, $content, $block ) {
+	if ( ! $block->context['postId'] ) {
+		return '';
+	}
+
+	$theme_post = get_post( $block->context['postId'] );
+	$theme = wporg_themes_theme_information( $theme_post->post_name );
+
+	$attributes = array(
+		'theme' => $theme->slug,
```

Comment:
Should this be escaped?

Reply:
Done for all the attributes in https://github.com/WordPress/wporg-theme-directory/pull/54/commits/15e3713c99b92e7013455b61e55841e262d0d7dc